src/Microsoft.OpenApi.Hidi/Program.cs

```diff
@@ -16,8 +16,19 @@ namespace Microsoft.OpenApi.Hidi
     static class Program
     {
         static async Task Main(string[] args)
-        {            
-            var rootCommand = new RootCommand() {};
+        {
+            var rootCommand = CreateRootCommand();
+
+            // Parse the incoming args and invoke the handler
+            await rootCommand.InvokeAsync(args);
+
+            //// Wait for logger to write messages to the console before exiting
+            await Task.Delay(10);
```

Comment:
Nope, When I ran into those issues with kiota it was usually because I was either not disposing a logging factory properly or because the command handler was using an overload that didn't use tasks, and wasn't waiting for completion of the task

Reply:
Ack.  I'll see if I can find the source of this issue.

Reply:
Actually, this has been an issue for a while now and is unrelated to this this PR.  Let's not hold up this PR further and solve this issue independently.  I've opened an issue to track it #1127